sdks/python/apache_beam/transforms/window.py

```diff
@@ -245,10 +255,16 @@ def __init__(self, value, timestamp):
     self.value = value
     self.timestamp = Timestamp.of(timestamp)
 
-  def __cmp__(self, other):
+  def __eq__(self, other):
+    return self.value == other.value and self.timestamp == other.timestamp
+
+  def __hash_(self):
```

Comment:
Why do you need to implement this? Also there is a typo in the name `__hash_`.

Reply:
I guess its not needed then :) I'll remove it.